.github/workflows/release.yml

```
@@ -0,0 +1,38 @@
+name: Release
+
+on: :
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 3: mapping values are not allowed in this context

(syntax-check)